docs/modules/ROOT/pages/websockets.adoc

```diff
@@ -4,20 +4,20 @@ include::./includes/attributes.adoc[]
 
 Using a chatbot in a WebSockets environment is quite common, which is why the extension provides a few facilities to make such usages as easy as possible.
 
-* 1. Start by adding the link:https://quarkus.io/guides/websockets-next-tutorial[quarkus-websockets-next] dependency to your `pom.xml` file:
-
+. Start by adding the link:https://quarkus.io/guides/websockets-next-tutorial[quarkus-websockets-next] dependency to your `pom.xml` file:
```

Comment:
This will make the list properly ordered without having to define the numbers.

Reply:
Good to know, thanks

Reply:
For this to work, you really need the `+` below to make sure the list is seen as one and not several separated lists.